src/Exception/InvalidArgumentException.php

```diff
@@ -12,14 +12,14 @@
 
 class InvalidArgumentException extends BaseInvalidArgumentException
 {
-    public static function oneOfCommandArguments()
+    public static function oneOfCommandArguments(): InvalidArgumentException
     {
         return new static(
             sprintf('One of the command arguments "%s" is required', join('", "', func_get_args()))
         );
     }
 
-    public static function unexpectedArgument($expectedType, $actualType, $position, $symbol)
+    public static function unexpectedArgument($expectedType, $actualType, $position, $symbol): InvalidArgumentException
```

Comment:
Missing parameter type hints, return type should be self

Reply:
this one is never used in the code, need to keep it ?